src/NLPModels.jl

```diff
@@ -275,6 +278,7 @@ end
 "Evaluate the product of the Lagrangian Hessian at `(x,y)` with the vector `v`."
 function hprod(nlp :: NLPModel, x :: Array{Float64}, y :: Array{Float64}, v :: Array{Float64})
   nlp.neval_hprod += 1
+  nlp.hv = zeros(size(x)) # JPD workaround 
```

Comment:
I'm not sure I understand this; wasn't `nlp.hv` already initialized to the vector of zeros at line 144?